PersonApi/V1/Gateways/DynamoDbGateway.cs

```diff
@@ -10,14 +12,19 @@ namespace PersonApi.V1.Gateways
     public class DynamoDbGateway : IPersonApiGateway
     {
         private readonly IDynamoDBContext _dynamoDbContext;
+        private readonly ILogger<DynamoDbGateway> _logger;
 
-        public DynamoDbGateway(IDynamoDBContext dynamoDbContext)
+        public DynamoDbGateway(IDynamoDBContext dynamoDbContext, ILogger<DynamoDbGateway> logger)
         {
             _dynamoDbContext = dynamoDbContext;
+            _logger = logger;
         }
 
+        [LogCall]
         public async Task<Person> GetPersonByIdAsync(Guid id)
         {
+            _logger.LogDebug($"Calling IDynamoDBContext.LoadAsync for id {id}");
```

Comment:
Are we not supposed to be using Lambda.logger()

Reply:
We are using the lambda logger, just through the standard Microsoft.Extensions.Logging abstractions.